checker/src/main/java/org/checkerframework/checker/units/UnitsRelationsDefault.java

```diff
@@ -5,7 +5,6 @@
 import org.checkerframework.checker.units.qual.Prefix;
 import org.checkerframework.checker.units.qual.g;
 import org.checkerframework.checker.units.qual.h;
-import org.checkerframework.checker.units.qual.kg;
```

Comment:
This one is interesting... Usually spotless removes unnecessary imports, right? Is the reason why this doesn't work here the field `kg` in this file?